conjur/policy/policy_controller.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+"""
+PolicyController module
+
+This module is the controller that facilitates all list actions
+required to successfully execute the POLICY command
+"""
+import sys
+
+
+class PolicyController:
+    """
+    PolicyController
+
+    This class represents the Presentation Layer for the POLICY command
+    """
+    def __init__(self, ssl_verify, policy_logic, policy_data):
```

Comment:
Add types to parameters

Reply:
would i also strong type the policy_logic and policy_data?

Reply:
it always good and not to hard to type all :)
But most importent a `ssl_verify : boolean` could make code more readble

Reply:
I will address this in my next PR